src/Codec/Archive/Zip.hs

```diff
@@ -416,9 +422,25 @@ writeSymbolicLinkEntry opts entry =
              let targetPath = fromJust . symbolicLinkEntryTarget $ entry
              let symlinkPath = prefixPath </> eRelativePath entry
              when (OptVerbose `elem` opts) $ do
-               hPutStrLn stderr $ "linking " ++ symlinkPath ++ " to " ++ targetPath
-             createSymbolicLink targetPath symlinkPath
+               let logLine = if symOpt == OptSymlinkClobber
+                     then "forcefully linking " ++ symlinkPath ++ " to " ++ targetPath
+                     else "linking: " ++ symlinkPath ++ " to " ++ targetPath
```

Comment:
Maybe for parallelism we should remove the colon in 427.

Reply:
oops, fixed